Gems/ROS2/Code/Source/RobotImporter/URDF/PrefabMakerUtils.cpp

```diff
@@ -105,10 +105,29 @@ namespace ROS2::PrefabMakerUtils
 
         if (auto* transformComponent = entity->FindComponent<AzToolsFramework::Components::TransformComponent>(); transformComponent)
         {
-            transformComponent->SetParent(parentEntityId);
+            if (!useLocalTransform)
+            {
+                transformComponent->SetParent(parentEntityId);
+            }
+            else
+            {
+                transformComponent->SetParentRelative(parentEntityId);
+            }
```

Comment:
Tiny suggestion: Positive logic is generally easier to read than negative logic, so consider flipping the if/else.

```suggestion
            if (useLocalTransform)
            {
                transformComponent->SetParentRelative(parentEntityId);
            }
            else
            {
                transformComponent->SetParent(parentEntityId);
            }
```

Reply:
Yes, I'll make that change